pkg/core/serving/common/common.go

```diff
@@ -25,28 +25,48 @@ import (
 	componentsv1alpha1 "github.com/dapr/dapr/pkg/apis/components/v1alpha1"
 	"github.com/go-logr/logr"
 	jsoniter "github.com/json-iterator/go"
-	yaml "gopkg.in/yaml.v3"
-	v1 "k8s.io/api/core/v1"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+	"gopkg.in/yaml.v3"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
 	openfunction "github.com/openfunction/apis/core/v1beta1"
+	"github.com/openfunction/pkg/constants"
+	"github.com/openfunction/pkg/core"
+	"github.com/openfunction/pkg/util"
 )
 
+// DaprSidecarMode is the inject mode for Dapr sidecar
+type DaprSidecarMode string
+
 const (
 	FunctionContextEnvName = "FUNC_CONTEXT"
 
 	ServingLabel        = "openfunction.io/serving"
+	ProxyLabel          = "openfunction.io/proxy"
 	OpenfunctionManaged = "openfunction.io/managed"
 
-	DaprEnabled     = "dapr.io/enabled"
-	DaprAppID       = "dapr.io/app-id"
-	DaprLogAsJSON   = "dapr.io/log-as-json"
-	DaprAppProtocol = "dapr.io/app-protocol"
-	DaprAppPort     = "dapr.io/app-port"
-	DaprMetricsPort = "dapr.io/metrics-port"
+	DaprEnabled           = "dapr.io/enabled"
+	DaprAppID             = "dapr.io/app-id"
+	DaprLogAsJSON         = "dapr.io/log-as-json"
+	DaprAppProtocol       = "dapr.io/app-protocol"
+	DaprAppPort           = "dapr.io/app-port"
+	DaprMetricsPort       = "dapr.io/metrics-port"
+	DaprListenAddresses   = "dapr.io/sidecar-listen-addresses"
+	DaprSidecarInjectMode = "dapr.io/sidecar-mode"
```

Comment:
DaprSidecarInjectMode => DaprServiceModeSidecar
The another mode should be `DaprServiceModeStandalone`